packages/circuit-ui/components/NotificationToast/NotificationToast.docs.mdx

```diff
@@ -0,0 +1,121 @@
+import { Status, Props, Story } from '../../../../.storybook/components';
+import { NotificationToast } from '@sumup/circuit-ui';
+
+# NotificationToast
+
+<Status.Stable />
+
+<Story id="notification-notificationtoast--base" />
+<Props />
+
+The notification toast component non-disruptively provides quick and contextual feedback to the user. It can be based on the outcome of an action (i.e. saving a form) or it can pop up without having been triggered explicitly (i.e. a toast message "You made a new sale" showing real time sales in an app).
+
+## When to use it
+
+The notification toast should be used for a feedback on micro actions (i.e. successfully/unsuccessfully saving some information).
+
+## Usage guidelines
+
+- Be concise, specific and short in the message and always provide a body copy. If needed an optional headline can be included above the body copy.
+- The notification toast floats above the content, positioned at the bottom of the screen, centralized horizontally within the content area.
+- For multiple stacked toasts, the new toast slides on top of the previous one. The timeout of the stacked toasts follows chronological order.
+- The auto-dismiss duration is adjustable (default minimum duration is 3000 ms).
+- Toasts can also be dismissed by clicking on a close button in the top-right corner.
```

Comment:
This is also obvious IMO.

Generally in other component docs I've been removing these "usage guidelines" sections altogether, they seem very "random" and subjective (e.g. why are we talking about the optional headline but not the different variants here? It doesn't seem consistent/systematic)

Feel free to leave it in here for now, but I think that it's more valuable to write this kind of things either under dedicated sections in "Accessibility" or in the component variations (e.g. variants)

Again, we can revisit this once we come up with a docs template that we can use consistently